### PR TITLE
WW-174 appreciation feature: add logging after send email event

### PR DESCRIPTION
**Pull request by dianafa**
https://wikia-inc.atlassian.net/browse/WW-174

### CODE REVIEW

File: extensions/wikia/ContributionAppreciation/ContributionAppreciationController.class.php [hunk 1]
```diff
@@ -25,6 +25,11 @@ public function appreciate() {
 			// send email that user received appreciation.
 			// Currently disabled because of: https://wikia-inc.atlassian.net/browse/WW-172
 			// $this->sendMail( $revisionId );
```
Comment (WikiaTech) [inline extensions/wikia/ContributionAppreciation/ContributionAppreciationController.class.php]: MAJOR Remove this commented out code. [rule](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS125)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 17 issues
* MAJOR 15 major
* MINOR 2 minor

Watch the comments in this conversation to review them.

#### Top 10 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. MAJOR [ContributionAppreciationController.class.php#L70](https://github.com/Wikia/app/blob/fa2c9437c1e7f26f68220e82a181b9b431a45945/extensions/wikia/ContributionAppreciation/ContributionAppreciationController.class.php#L70): Use "static" keyword instead of "self". [rule](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS2037)
1. MAJOR [ContributionAppreciationController.class.php#L87](https://github.com/Wikia/app/blob/fa2c9437c1e7f26f68220e82a181b9b431a45945/extensions/wikia/ContributionAppreciation/ContributionAppreciationController.class.php#L87): Use "static" keyword instead of "self". [rule](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS2037)
1. MAJOR [ContributionAppreciationController.class.php#L95](https://github.com/Wikia/app/blob/fa2c9437c1e7f26f68220e82a181b9b431a45945/extensions/wikia/ContributionAppreciation/ContributionAppreciationController.class.php#L95): Use "static" keyword instead of "self". [rule](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS2037)
1. MAJOR [ContributionAppreciationController.class.php#L105](https://github.com/Wikia/app/blob/fa2c9437c1e7f26f68220e82a181b9b431a45945/extensions/wikia/ContributionAppreciation/ContributionAppreciationController.class.php#L105): Use "static" keyword instead of "self". [rule](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS2037)
1. MAJOR [ContributionAppreciationController.class.php#L107](https://github.com/Wikia/app/blob/fa2c9437c1e7f26f68220e82a181b9b431a45945/extensions/wikia/ContributionAppreciation/ContributionAppreciationController.class.php#L107): Use "static" keyword instead of "self". [rule](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS2037)
1. MAJOR [ContributionAppreciationController.class.php#L109](https://github.com/Wikia/app/blob/fa2c9437c1e7f26f68220e82a181b9b431a45945/extensions/wikia/ContributionAppreciation/ContributionAppreciationController.class.php#L109): Use "static" keyword instead of "self". [rule](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS2037)
1. MAJOR [ContributionAppreciationController.class.php#L118](https://github.com/Wikia/app/blob/fa2c9437c1e7f26f68220e82a181b9b431a45945/extensions/wikia/ContributionAppreciation/ContributionAppreciationController.class.php#L118): Use "static" keyword instead of "self". [rule](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS2037)
1. MAJOR [ContributionAppreciationController.class.php#L134](https://github.com/Wikia/app/blob/fa2c9437c1e7f26f68220e82a181b9b431a45945/extensions/wikia/ContributionAppreciation/ContributionAppreciationController.class.php#L134): Reduce the number of conditional operators (5) used in the expression (maximum allowed 3). [rule](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1067)
1. MAJOR [ContributionAppreciationController.class.php#L139](https://github.com/Wikia/app/blob/fa2c9437c1e7f26f68220e82a181b9b431a45945/extensions/wikia/ContributionAppreciation/ContributionAppreciationController.class.php#L139): Use "static" keyword instead of "self". [rule](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS2037)
1. MAJOR [ContributionAppreciationController.class.php#L155](https://github.com/Wikia/app/blob/fa2c9437c1e7f26f68220e82a181b9b431a45945/extensions/wikia/ContributionAppreciation/ContributionAppreciationController.class.php#L155): Use "static" keyword instead of "self". [rule](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS2037)

**Comment by sebastianmarzjan** [comment]
LGTM

**Comment by cormorando** [comment]
LGTM

**Pull request merged**